### ISSUE TITLE
No cells as valid with Beehive Scanning and Geofences

### ISSUE THREAD

**Issue by Kallo1994** [RocketMap/RocketMap#2530]
## Expected Behavior
Normally geofences should be used to limit the search radius and should work with Beehive Scanning.

## Current Behavior
I testet it with 30 running accounts, there are two active geofences, the location is in one of the active geofences and speed-scan is enabled.
Without Beehive Scanning it works great, but with Beehive Scanning I get the following message:
"No cells regarded as valid for desired scan area. Check your provided geofences. Aborting."

**Comment by loosi** [comment]
I can confirm the same behaviour. Valid location with speed-scan and invalid location with beehive.

**Comment by mauricionarcizo** [comment]
The same problem here.

**Comment by sebastienvercammen** [comment]
@loosi @mauricionarcizo 👍 Thanks for confirming, we're looking into it.

**Comment by xLeViet** [comment]
So i cant have selected area witg geofences and using beehive searching method?

**Comment by sebastienvercammen** [comment]
@Kallo1994 @loosi @mauricionarcizo Can you include logs for your instances? I'm specifically interested in:
```
Using matplotlib: %s.
Found %d coordinates to geofence.
Geofenced to %s coordinates in %.2fs.
```
and a link to a `--dump` result please.

**Comment by loosi** [comment]
`Using matplotlib: False.
Found 2611 coordinates to geofence.
Geofenced to 352 coordinates in 0.20s.`

https://hastebin.com/yopugexili.txt

**Comment by ghost** [comment]
Using matplotlib: False. Found 61 coordinates to geofence. Geofenced to 54 coordinates in 0.01s.

Normal Log: https://hastebin.com/asemotusiv.md
Dump: https://hastebin.com/evaduhidow.txt

**Comment by sebastienvercammen** [comment]
@Kallo1994 @loosi Thanks for the details!

Could you please check that **all** of your hives are covered by your geofence? It seems one of your hives may be entirely outside of the geofence, which is why this happens:
```
2018-03-15 18:59:29,151 [   search-overseer][      geofence][    INFO] Geofenced to 0 coordinates in 0.00s.
2018-03-15 18:59:29,151 [   search-overseer][    schedulers][   ERROR] No cells regarded as valid for desired scan area. Check your provided geofences. Aborting.
```

@loosi I assume you used `--dump` on an edited config (which you probably did to get it running?). Because:
```
 'beehive': False,
```

**Comment by ghost** [comment]
@sebastienvercammen It may well be that one or more of the hives are outside the geofence. But I would assume that geofence cares about it, ignores it and only launches the hives that are inside.

**Comment by loosi** [comment]
@sebastienvercammen Yes that was the working config without beehive. I tried it again, but nothing has changed in code since then. Im using one geofence right now, an that position ist inside that fence

https://hastebin.com/yuzuqujimu.txt

```
search-overseer][      geofence][    INFO] Loading geofenced or excluded areas 
search-overseer][      geofence][    INFO] Loaded 1 geofenced and 1 excluded areas
search-overseer][        search][    INFO] Enabling new API force Watchdog.
search-overseer][        search][    INFO] New location caught, moving search grid.
search-overseer][      geofence][    INFO] Using matplotlib: False.
search-overseer][      geofence][    INFO] Found 2611 coordinates to geofence.
search-overseer][      geofence][    INFO] Geofenced to 428 coordinates in 0.15s.
search-overseer][    schedulers][    INFO] 428 steps created
search-overseer][    schedulers][    INFO] Initial spawnpoint scan is complete
search-overseer][    schedulers][    INFO] Found 2049 spawn points within hex
search-overseer][    schedulers][    INFO] Doing 876,972 distance calcs to assign spawn points to scans
search-overseer][    schedulers][    INFO] 2060 relations found between the spawn points and steps
search-overseer][      geofence][    INFO] Using matplotlib: False.
search-overseer][      geofence][    INFO] Found 2611 coordinates to geofence.
search-overseer][      geofence][    INFO] Geofenced to 0 coordinates in 0.08s.
search-overseer][    schedulers][   ERROR] No cells regarded as valid for desired scan area. Check your provided geofences. Aborting.
```

**Comment by dannythunder** [comment]
In geofence, its the "most outer" thats is relevant here (light grey).

This is my setup. I could only scan behive-mode, with geofence, at -st 6 at most.

Same log as ppl above me.

<img width="268" alt="skarmavbild 2018-03-17 kl 01 53 19" src="https://user-images.githubusercontent.com/1385484/37549985-81c8f7d8-2986-11e8-9985-ab3b2e82a15f.png">
<img width="266" alt="skarmavbild 2018-03-17 kl 01 53 25" src="https://user-images.githubusercontent.com/1385484/37549986-81e62cc2-2986-11e8-94ce-9f0df047750f.png">

**Comment by dannythunder** [comment]
note*

same for exclude.
